<?xml version="1.0" encoding="utf-8"?>
<launch>
  <rosparam command="load" file="$(env HOME)/tdmms_ws/devel/tdmms_parameter.yaml"/>
  <node pkg="adv_dio" name="adv_dio_node" type="adv_dio_node"/>
  <node pkg="tdmms_transfer_master" name="tdmms_transfer_master" type="tdmms_transfer_master"/>
  <node pkg="sample_stage_valve" name="sample_stage_valve" type="sample_stage_valve"/>
  <node pkg="arm_transfer_valve" name="arm_transfer_valve" type="arm_transfer_valve"/>
  <node pkg="iai_gripper" name="iai_gripper_node" type="iai_gripper_node"/>
  <node pkg="tape_stage_valve" name="tape_stage_valve" type="tape_stage_valve" />
</launch>
